### DISCUSSION TITLE
Notifications Tab Doesn’t Update Automatically

### DISCUSSION THREAD

**Discussion by mikaelacaron** [community/community#69488] [Mobile] [answered]
I would've expected when I first open the Home tab, it loads my repos, and then switching to the Notifications tab, it also update to the latest data, but I switch to the Notifications tab, and nothing happens, I need to pull to refresh to see the latest notifications

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.133.1 (112099449)
Device: iPhone15,2 (iOS 17.0.3)
</details>

**Comment by stevepopovich** [comment, marked as answer]
Hi @mikaelacaron thanks for the feedback. This is definitely something we can do better. I have passed this along to the team for consideration.

**Reply by mikaelacaron** [reply]
I just saw an update with version `1.134.0` that mentions "prevent Inbox from showing stale notifications on first load" but I'm still seeing this happening

**Reply by stevepopovich** [reply]
Is this Android or iOS?

**Reply by mikaelacaron** [reply]
iOS!

**Reply by stevepopovich** [reply]
Thank you!
